include/deal.II/hp/dof_handler.h

```diff
@@ -1120,6 +1224,25 @@ namespace hp
      */
     std::map<const cell_iterator, const unsigned int> coarsened_cells_fe_index;
 
+#ifdef DEAL_II_WITH_P4EST
+    /**
```

Comment:
Let's not do this `#ifdef` here. If deal.II wasn't configured with p4est, then these member variables will simply not ever be used. But at least the size of the object is always the same.

Reply:
I haven't looked at it from this perspective. I agree.